### DISCUSSION TITLE
PocketHost 0.7.0 released

### DISCUSSION THREAD

**Discussion by benallfree** [pockethost/pockethost#217] [Show and tell]
[PocketHost](https://pockethost.io), the zero-config 'PocketBase in the cloud' platform, has reached v0.7.0 today. Thank you for everyone's continued support for this project.

## Stats

* 317 github stars (personal best!)
* ~2000 registered users
* ~2500 instances created
* ~1.5 million instance invocations to date (# times PocketBase executable has been launched)
* ~400 instances active and used in the past week. This tracks, about 15% of instances are in active use. 

## FAQ
* **Why is this free?** Because I feel like it.
* **Why isn't there a paid tier?** Because I don't feel like it. Maybe someday there will be a donation/patron tier, but that's about it.
* **But who really pays for it?** I do. Completely. I created an endowment from personal funds to keep PocketHost running for at least 5 years.
* **What about uptime, SLA, etc?** No promises. But it's working for a lot of people, as you can see!
* **Is PocketHost a business? Does it have investors?** No, it's a personal project and will stay that way. If it turns into anything, it will be a nonprofit. Investors have contacted me, but I am not interested in that path. I want this to stay grassroots, organic, and user-supported.
* **How do I update my instance to the latest version of PocketBase** For now, start a discussion in the PH discussion forum or send an email to ben@pockethost.io
* **I have an idea/feature request** Start a new discussion and then start a PR :)

## About this release

This is a maintenance release.

**PocketBase Version Management**

Previously, PocketHost locked PocketBase to a known safe version. Today, I feel it's safe and stable enough to allow new instances and the PocketHost central database to always use the latest version of PocketBase. 

As new PocketBase versions are released, they are automatically adopted as the newest version to use.

**Native PocketBase Migrations**

Now that PocketBase supports migrations natively, we use those to maintain the central database schema.

## The future

We've moved everything into a [roadmap document](https://github.com/benallfree/pockethost/blob/master/roadmap.md) that you can check out. Help us prioritize and build these features!

**Comment by bloor** [comment]
Thanks for the hard work ✌️

**Comment by DustinDust** [comment]
You are the goat! TYSM for powering my first ever (real) personal project

**Reply by benallfree** [reply]
When you're ready, please share it in the #180 thread. At some point, we're going to make a showcase page of apps powered by PH :)
